content/posts/newsletter/2019-10-01.md

```diff
@@ -0,0 +1,124 @@
+
+<form style="padding: 10px; text-align:center; margin-bottom: 30px;"
+      action="https://tinyletter.com/xmpp" method="post" target="popupwindow"
+      onsubmit="window.open('https://tinyletter.com/xmpp', 'popupwindow',
+      'scrollbars=yes,width=800,height=600');return true">
+<p><label for="tlemail">Subscribe to receive the next edition in your inbox</label></p>
+<p><input type="text" placeholder="Email address" name="email" id="tlemail" /></p>
+<input type="hidden" value="1" name="embed"/>
+<input type="submit" style="padding: 10px; border-radius: 5%" value="Subscribe" />
+</form>
+
+## Articles
+
+Ralph Meijer, chairman of the Board of Directors of the XSF, has written an introductory piece about "[XMPP Message Attaching, Fastening, References](https://ralphm.net/blog/2019/09/09/fastening)", specifications currently in progress.
+
+In case of internet shut down, whether by disaster or voluntarily, peer networks are useful. [Monal uses Airdrop](http://monal.im/blog/xmppsignalbluetoothp2p-wifi-serverless-chat/) (wifi and bluetooth) along with XMPP and OMEMO end-to-end encryption.
+
+![Monal and Airdrop screenshots](https://framapic.org/Qg1mv43w1673/akPmtepp0nfH.png)
+
+JabberFr, the French XMPP/Jabber community, has again [translated the XMPP newsletter in French](https://news.jabberfr.org/2019/09/lettre-dactualite-xmpp-du-3-septembre-2019/), merci beaucoup.
+
+We have started a very minimalistic [communication guide](https://wiki.xmpp.org/web/Basic_communication_guide_for_XMPP_techies) to help promoting a project through social networks and other means such as blog posts. Could be worth for those involved in XMPP projects and would like to get ideas on how to reach out to different communities.
+
+## Events
+
+As usual every year, that time has come: we proudly announce the XMPP Summit 24 and [FOSDEM](https://fosdem.org/2020/). The 24th XMPP Summit will happen on Thursday 30th and Friday 31st of January, and FOSDEM will be held on Saturday 1st and Sunday 2nd of February. Prepare for the community gathering; now is good time to start booking your flights!
+
+![FOSDEM 2020](https://framapic.org/fTZlikXsyi0F/ZqlvJhNjWDFj.png)
+
+## Software releases
+          
+### Servers
+
+Réda Housni Alaoui is reviving the formerly dormant [Vysper XMPP server](https://github.com/apache/mina-vysper) (pronounced as "whisper").
+
+
+[Xabber Server v.0.9 alpha is released](https://blog.xabber.com/xabber-server-v-0-9-alpha-is-released/), with quick installation and management panel and interesting innovations.
+
+[MongooseIM 3.4.1 is out now](https://www.erlang-solutions.com/resources/download.html) with an important security upgrade, fixing a vulnerability allowed any logged in user to crash the node with malicious stanza on certain (but popular) configurations. [Read the whole thread on Twitter](https://twitter.com/MongooseIM/status/1176111308430815232) for more information.
+
+The igniterealtime community has many news:
```

Comment:
You can't say 'many news'. Use “much” or “a lot of / lots of”

Reply:
OK, fixed